Classes/Helpers/OpencastHelper.php

```diff
@@ -90,11 +127,66 @@ public function getPreviewImage(File $file)
      * @return array with metadata
      */
     public function getMetaData(File $file)
+    {
+        $mediaId = $this->getOnlineMediaId($file);
+        $metadata = $this->fetchMetaData($mediaId);
+
+        return $metadata;
+    }
+
+    protected function getTitle($mediaId): string
+    {
+        return $this->fetchMetaData($mediaId)['title'];
+    }
+
+    protected function fetchMetaData($mediaId): array
     {
         $metadata = [];
 
-        $metadata['title'] = $file->getProperty('title');
+        if ($data = $this->fetchJson($mediaId)) {
+            $metadata['title'] = $data['dcTitle'];
```

Comment:
is the metadata documented somewhere (URL)? Maybe having a reference as a comment here would help to understand where these properties come from. Or did you just found them out by empirical testing with some samples?

Reply:
@baschny, just added PHP comment with a link to these docs: https://stable.opencast.org/docs.html?path=/search#episodes-1